management/mongodb_test.go

```diff
@@ -124,7 +128,7 @@ func TestAddMongoDB(t *testing.T) {
 		})
 		assert.NoError(t, err)
 		assert.NotNil(t, serviceOK)
-		assert.Equal(t, services.GetServiceOKBody{
+		assert.Equal(tt, services.GetServiceOKBody{
```

Comment:
I think using `t` and `tt` in the same test would be too confusing.

Let's rename `t` to `realT` or something, define `t` with `pmmapitests.ExpectFailure` and use it in the whole test. When this test starts to pass, we will just remove `pmmapitests.ExpectFailure` and rename `realT` to `t`.

Reply:
yes, it might be confusing, but we expect that only some asserts should fail and others shouldn't.

Reply:
Mm, maybe we could make smaller tests then? It would be better to have test binary fail/pass, not "some assertions pass, some are failed as expected, and some other fail in unexpected way".